.gitignore

```diff
@@ -67,5 +70,10 @@ typings/
 __pycache__
 build
 *.egg-info
+setup.pye
+
+# Python virtual env
+venv
```

Comment:
why adding venv?

Reply:
Because it's a natural practice to install dependencies under venv during development.